homepage/templates/homepage/reviews/single_review.html

```diff
@@ -0,0 +1,64 @@
+{% load static %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/single_review.css' %}">
+</head>
+<body>
+<div class="review-box">
+    <div class="title-row">
+        <h4>{{review.user}}</h4>
+        <h6>{{review.date}}</h6>
+    </div>
+    <div>
+        About Course: {{review.course.name}}
+    </div>
+    <div style="margin-top: 20px">
+        {% if review.rate == 5 %}
+        {% for i in "12345"|make_list %}
+        <img src="{% static 'icons/star.png' %}">
+        {% endfor %}
+        {% elif review.rate == 4 %}
+        {% for i in "1234"|make_list %}
+        <img src="{% static 'icons/star.png' %}">
+        {% endfor %}
+        {% elif review.rate == 3 %}
+        {% for i in "123"|make_list %}
+        <img src="{% static 'icons/star.png' %}">
+        {% endfor %}
+        {% elif review.rate == 2 %}
+        {% for i in "12"|make_list %}
+        <img src="{% static 'icons/star.png' %}">
+        {% endfor %}
+        {% else %}
+        <img src="{% static 'icons/star.png' %}">
+        {% endif %}

```

Comment:
maybe you can do it simpler something like:
https://stackoverflow.com/a/17691875
```suggestion
        {% for i in range(review.rate) %}
        <img src="{% static 'icons/star.png' %}">
        {% endfor %}
```

Reply:
@kobihk I tried it but I get a template syntax error.
According to this discussion https://stackoverflow.com/questions/1107737/numeric-for-loop-in-django-templates
and this bug report https://code.djangoproject.com/ticket/5172, it looks like Django does not support it. The link you've attached is referring to Jinja.

Reply:
@ellaml please try with this code:
        {% for i in " "|rjust:review.rate|make_list %}
        <img src="{% static 'icons/star.png' %}">
        {% endfor %}